RestaurantReservation.Db/Customers.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace RestaurantReservation.Db
+{
+    public class Customers
```

Comment:
actually, singular table names is preferred, can we rename this to **Customer**, the same applies for other tables, pls check them also